pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md

```diff
@@ -0,0 +1,117 @@
+description: 'Ensure your Web Stories are reaching your intended audience by including necessary metadata and markup.'
+author: CrystalOnScript
+---
+
+Ensure your Web Stories are reaching your intended audience by including necessary metadata and markup.
+
+
+# AMP markup
+
+Web Stories with missing or incorrect AMP markup may not surface in search engines or third party platforms. All Web Stories must include a canonical URL that points to the Web Story itself. 
+
+
+```html
+<link rel="canonical" href="https://www.example.com/url/to/Web/Story.html">
+```
+
+
+
+## Required &lt;amp-story&gt; attributes
+
+Web Stories must include the following [`<amp-story>`](https://amp.dev/documentation/components/amp-story)attributes. The `<amp-story>` component must be the only child element of `<body>`.
+
+```html
+<body>
+    <amp-story standalone 
+    title="Web Story Title"
+    publisher="Web Stor Publisher"
+    publisher-logo-src="https://example.com/logo/1x1.png"
+    poster-portrait-src="https://example.com/my-story/poster/3x4.jpg">
+    ...
+    </amp-story-standalone>
+</body>
+```
+
+
+### `standalone`
+
+Identifies the AMP document is a Web Story. 
+
+### `title`
+
+The title of the Web Story. 
+
+### `publisher`
+
+The name of the Web Story publisher.
+
+
+### `publisher-logo-src`
+
+A URL to the Web Story publisher's logo image. The logo image should be larger than or equal to 96px by 96px and maintain a 1:1 aspect ratio.
+
+
+### `poster-portrait-src`
+
+A URL to an image used as the [Web Story poster](https://amp.dev/documentation/components/amp-story/#poster-guidelines-(for-poster-portrait-src,-poster-landscape-src,-and-poster-square-src)). Used as the cover for the Web Story and should be representative of the story. Do not pre-embed the Web Story title on it. Should be 853px by 640px and maintain a 3:4 aspect ratio.
+
+
+## Recommended <amp-story> attributes
+
+### `poster-landscape-src`
+
+A URL to an image used as the [Web Story poster](https://amp.dev/documentation/components/amp-story/#poster-guidelines-(for-poster-portrait-src,-poster-landscape-src,-and-poster-square-src)) in landscape format. Used as the cover for the Web Story and should be representative of the story. Do not pre-embed the Web Story title on it. Should be 640 px by 853 px and maintain a 4:3 aspect ratio.
+
+### `poster-square-src`
+
+A URL to an image used as the [Web Story poster](https://amp.dev/documentation/components/amp-story/#poster-guidelines-(for-poster-portrait-src,-poster-landscape-src,-and-poster-square-src)). Used as the cover for the Web Story and should be representative of the story. Do not pre-embed the Web Story title on it. Should be 640 px by 640 px and maintain a 1:1 aspect ratio.
+
+
+### `supports-landscape`
+
+Allows a Web Story viewing on a mobile device held in a landscape orientation and changes the desktop experience to an immersive full-beed mode. Removes the default three portrait panel experience on desktop. 
+
+
+# HTML markup
```

Comment:
I think these can also probably moved into the html metadata section, they technically all are